public/components/datasources/components/manage/accelerations/acceleration_table.tsx

```diff
@@ -242,8 +235,8 @@ export const AccelerationTable = ({ dataSourceName }: AccelerationTableProps) =>
       field: 'refreshType',
       name: 'Refresh Type',
       sortable: true,
-      render: (autoRefresh: boolean, acceleration: CachedAccelerations) => {
-        return <EuiText>{acceleration.autoRefresh ? 'Auto refresh' : 'Manual'}</EuiText>;
+      render: (_, acceleration: CachedAccelerations) => {
```

Comment:
The underscore  `_ ` is used to signify that the first parameter is intentionally ignored, since `EuiInMemoryTable` expects the first parameter to be the value of the field specified by `field`, and the second parameter to be the entire row object